index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone Project</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+
+  <header>
+    <div class="main-nav-section">
+      <div class="navbar-container">
+        <ul class="navbar font-lato">
+          <li>
+            <i class="fab fa-facebook-f" aria-hidden="true"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter" aria-hidden="true"></i>
+          </li>
+          <li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="secondary-navbar">
+      <div class="secondary-sub-container">
+        <div class="navbar-logo">
+          <a class="font-lato" href="index.html">CCS SUMMER CONCERT <br> Series 2021</a>
+        </div>
+        <!-- open and close icons -->
+        <i class="fas fa-bars fa-3x text-white cursor-pointer" id="openMenu" aria-hidden="true" ></i>
+
+        <nav class="pages-menu" id="mob-nav">
+          <i class="fa fa-times fa-3x text-white" aria-hidden="true" id="closeMenu"></i>
+          <ul class="navbar-items">
+            <li class="page-sections font-lato home-link">
+              <a href="index.html" class="active-tab">Home</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="about.html">About</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Program</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Join</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="product">
+            <a class="font-lato" href="index.html">CCS Concert</a>
+          </div>
+        </nav>
+      </div>
+    </div>
+  </header>
```

Comment:
The top dark navbar and the bottom white navbar of your desktop version do not match the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly note we actually want our design to as much as possible _mimic_ the design of the provided template design. Kindly ensure to improve upon these aspects:
- [ ] The length of the top dark navbar is **shorter**, as can be seen in the original design below.
- [ ] The contents on both navbars are **centered** and that that the _padding_ of the extreme left and rights sides of the nav contents are equal

**Your header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/132956294-986eb314-cf3e-4c7d-a25e-350aff0265dd.png)
<br>

**The original header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/132956330-393c697b-60c8-4a14-a9a8-0f7397ac04c9.png)